client/me/purchases/titles.js

```diff
@@ -10,9 +10,8 @@ export default {
 	addCreditCard: i18n.translate( 'Add Credit Card' ),
 	cancelPurchase: i18n.translate( 'Cancel Purchase' ),
 	confirmCancelDomain: i18n.translate( 'Cancel Domain' ),
-	editCardDetails: i18n.translate( 'Edit Card Details', { comment: 'Credit card' } ),
+	editCardDetails: i18n.translate( 'Change Card', { comment: 'Credit card' } ),
```

Comment:
I would go with "Change Card Details" here probably, for consistency if nothing else.

Reply:
@DavidRothstein "Change Card Details" implies that you can change one detail about the card and that the rest stays the same.
The whole purpose of this PR is that this is impossible.
You can only change the whole card and you are informed about this at the bottom of the form.
Also - what exactly we would be consistent with?

"Card Details" is indeed an axiom that we in payments are familiar with, but people outside of our world are not.
I'm happy to oblige, but there was reasoning in this phrasing :)

Reply:
>Also - what exactly we would be consistent with?

With "Add Card Details" (in the line of code below this one), which is what is used when adding a card to a subscription that doesn't have a payment method.

I see your point, but something about "Change Card" just seems wrong to me... maybe "Change Credit Card" would be better.  But I'm still not sure.

(Alternatively, we could just reuse "Add Card Details" -- or "Add Credit Card" -- in this case also?  Regardless of how they got to this page, the thing they are doing here is the same: adding a new credit card.)

Reply:
"Add Credit Card" implies that this will be additional card in some manner - "added one". It does not imply that this will be the one that will be billed.
"Change Credit Card" seems the most sane.

Reply:
>"Change Credit Card" seems the most sane.

OK, let's go with that.

I'll consider creating a followup PR to change the "Add Card Details" to "Add Credit Card", so they are consistent (and you are right that "Details" is a bit of a technical term).

Reply:
>I'll consider creating a followup PR to change the "Add Card Details" to "Add Credit Card", so they are consistent (and you are right that "Details" is a bit of a technical term).

Thanks for taking care of that in this PR already :)